@here/harp-datasource-protocol/lib/TileInfo.ts

```diff
@@ -249,6 +293,36 @@ export class ExtendedTileInfo implements TileInfo {
             this.typeCatalog = new Array<string>();
         }
     }
+
+    /**
+     * Compute the memory footprint caused by objects owned by the `ExtendedTileInfo`.
+     */
+    getNumBytes(): number {
+        let numBytes = 100;
```

Comment:
Why do you start with a memory consumption of 100 bytes?

Reply:
Every Object is a map that uses a lot of bytes for itself, and then add its properties. An object that actually takes up just 100 bytes would not be very big...

Reply:
Added constants